sdk/storage/inc/blobs/block_blob_client.hpp

```diff
@@ -124,7 +124,7 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @param options Optional parameters to execute this function.
      * @return A BlobContentInfo describing the state of the updated block blob.
```

Comment:
BlobContentInfo changed to UploadBlockBlobResult

Reply:
Ditto for other comments.

Reply:
fixed